pkgs/games/dwarf-fortress/unfuck.nix

```diff
@@ -33,6 +33,10 @@ let
       sha256 = "1kszkb1d1vll8p04ja41nangsaxb5lv4p3xh2jhmsmipfixw7nvz";
     };
     "0.47.01" = {
+      unfuckrelease = "0.47.01";
```

Comment:
```suggestion
      unfuckRelease = "0.47.01";
```

to fix evaluation

Reply:
r -> R

Reply:
Aha!! Good catch, on my phone right now but I'll confirm tonight.